base/docs/basedocs.jl

```diff
@@ -1999,6 +1999,39 @@ for bit in (8, 16, 32, 64, 128)
     end
 end
 
+"""
+	$(Int) <: Signed
```

Comment:
```suggestion
	Int <: Signed
```
Isn't more reasonable if we hardcode this?